practiceapp/src/main/java/com/practiceapp/practiceapp/service/PostService.java

```diff
@@ -24,9 +25,9 @@
     @Autowired
     private PlacesApi placesApi;
 
-    public PostEntity updatePost(PostEntity postEntity){
-        PostEntity post = postRepository.save(postEntity);
-        return post;
+    public int updatePost(PostEntity postEntity) {
+        postRepository.save(postEntity);
+        return 1;
```

Comment:
I don't understand why it returns 1. If you don't want to return post entity, maybe you can return a boolean and True or String and "success".